scripts/lint

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+function checkPristine() {
+    if [ -n "$(git status --porcelain)" ]; then
+        echo "$1"
+        exit 1
+    fi
+}
+
+function clangFormat() {
+    brew list clang-format@14 || brew install clang-format@14
+    find ./Backpack -name "*.[hm]" -exec clang-format -i {} \+ 
+}
+
+echo "Checking that the working tree is clean"
+checkPristine "You have uncommited changes. Please commit or stash them."
+clangFormat
+checkPristine "Formatter failed to format some files. Please run 'brew install clang-format@14' and try again."
+
+bundle exec pod lib lint --allow-warnings
```

Comment:
Should we execute lib lint for all 3 podspecs? I'm not sure which one it will lint right now